tests/rptest/tests/schema_registry_test.py

```diff
@@ -1013,3 +1014,30 @@ def test_protobuf(self):
         self.logger.info(result_raw)
         assert result_raw.status_code == requests.codes.ok
         assert result_raw.json() == [2]
+
+    @cluster(num_nodes=3)
+    def test_serde_client(self):
+        """
+        Verify basic serialization client
+        """
+        protocols = [SchemaType.AVRO, SchemaType.PROTOBUF]
```

Comment:
Nice idea. It's a bit of a trade-off with execution time. Happy to give it a go.

Reply:
> This could be a good candidate for the @matrix decorator in ducktape that way you don't need a loop over 2 values.

yeh

> Nice idea. It's a bit of a trade-off with execution time. Happy to give it a go.

another benefit of using `@matrix` is that the test framework treats the points parameter space as separate tests. so if one particular point fails ducktape can report the test configuration rather than forcing dev to go look through logs to see which configuration failed.